src/lib/notifications/backgroundActions.native.js

```diff
@@ -66,26 +67,35 @@ export const useNotifications = navigation => {
   useEffect(() => {
     // eslint-disable-next-line require-await
     const onClaimNotification = async navigation => navigation.navigate('Claim')
-
-    const subscription = Notifications.events().registerNotificationOpened(
-      async (notification, completion) => {
-        const navigation = getNavigation()
-        const { category } = notification?.payload || {}
-
-        switch (category) {
-          case NotificationsCategories.CLAIM_NOTIFICATION:
-            await onClaimNotification(navigation)
-            break
-          default:
-            break
-        }
-
-        completion()
-      },
-    )
+    const events = Notifications.events()
+    
+    const onForeground = (notification, completion) => {
+      log.info(`Notification received in foreground: ${notification.title} : ${notification.body}`)
+      completion({ alert: false, sound: false, badge: false })
```

Comment:
```suggestion
      // should call completion otherwise notifications won't receive
      completion({ alert: false, sound: false, badge: false })
```